src/engine/bank01.asm

```diff
@@ -7172,7 +7172,45 @@ Func_6cfa: ; 6cfa (1:6cfa)
 	ret
 ; 0x6d3f
 
-	INCROM $6d3f, $6d84
+Func_6d3f: ; 6d3f (1:6d3f)
+	or a
+	bit POISONED_F , [hl]
+	ret z
+	push hl
+	bit DOUBLE_POISONED_F, [hl]
+	ld a, PSN_DAMAGE
+	ldtx hl, Received10DamageDueToPoisonText    ; copies from the byte of the damage due to poison by 10
+	jr z, .Damage_Dealt_From_Poison
+	ld a, $14
+	ldtx hl, Received20DamageDueToPoisonText     ; copies from the byte of the damage due to poison by 20
+.Damage_Dealt_From_Poison
```

Comment:
Typically a local label branch name should be based on how it's used or why it might be taken or skipped.
Also, local labels (labels beginning with a `.`) should start with a lowercase letter.
In this case, I think a better name would be `.not_double_poisoned`, because the jump is taken if the Pokemon is only regular poisoned (the `DOUBLE_POISONED_F` flag is `0`).